modules/webbrowser/cmake/patch/cmake/cef_variables.cmake

```
@@ -253,15 +263,15 @@ if(OS_MACOSX)
     -Wno-unused-parameter           # Don't warn about unused parameters
     )
   list(APPEND CEF_C_COMPILER_FLAGS
-    -std=c99                        # Use the C99 language standard
+    -std=c17                        # Use the C99 language standard
     )
   list(APPEND CEF_CXX_COMPILER_FLAGS
     #-fno-exceptions                 # Disable exceptions
     #-fno-rtti                       # Disable real-time type information
     -fno-threadsafe-statics         # Don't generate thread-safe statics
     -fobjc-call-cxx-cdtors          # Call the constructor/destructor of C++ instance variables in ObjC objects
     -fvisibility-inlines-hidden     # Give hidden visibility to inlined class member functions
-    -std=gnu++14                    # Use the C++14 language standard including GNU extensions
+    -std=gnu++2a                    # Use the C++14 language standard including GNU extensions
```

```suggestion
    -std=gnu++20                    # Use the C++20 language standard including GNU extensions
```

it was called `2a` in GCC 9 and below, which we don't support anyway

---

Thanks @alexanderbock - I'd read at https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html about "decay" behaviour if a compiler doesn't support a particular flag, hence I used an earlier flag, because XCode typically lags behind in feature support. But it looks like XCode does support -std=gnu++20 from version 14 or so, https://developer.apple.com/documentation/xcode/build-settings-reference